File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        setA = set(nums)
        res = []
        for i in range(1, len(nums) + 1):
            if i in setA:
                continue
            else:
                res.append(i)
        return res

        

File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {

        HashSet<Integer> numSet = new HashSet<>();
        ArrayList<Integer> numList = new ArrayList<>();

        for (int num : nums) {
            numSet.add(num);
        }
        for(int i = 1 ; i < nums.length + 1 ; i ++){
            if (numSet.contains(i)){
                continue;
            }
            else{
                numList.add(i);
            }
        }
        return numList;
    }
}